src/main/java/cz/startnet/utils/pgdiff/schema/PgConstraint.java

```diff
@@ -56,7 +56,6 @@ public String getCreationSQL() {
         sbSQL.append(PgDiffUtils.getQuotedName(getTableName()));
         sbSQL.append(System.getProperty("line.separator"));
         sbSQL.append("\tADD CONSTRAINT ");
-        sbSQL.append(PgDiffUtils.getCreateIfNotExists());
```

Comment:
why did we remove this line?

Reply:
Add constraint doesn't support add if not exists, only drop if exists

Reply:
The constraint code is in the wrong PR, I will remove it because  there are the PR #226 with the same fix